### PR TITLE
SCSS cleanup

### PR DESCRIPTION
**Pull request by dacook**
#### What? Why?

A fix to prevent global variables becoming overwritten, and just lots of general tidying up.
Originally prompted by https://openfoodnetwork.slack.com/archives/C01CXQNJ1J6/p1687252447504379

It's almost entirely just moving/renaming variables.

- The only colour change is a very slight change of red on the modal close button (eg upload product photo on the bulk edit screen).
- Everything else should look the same when admin_style_v3 feature toggle is turned on or off. 

#### What should we test?
I think the main test is that the SCSS compiler succeeded; there shouldn't need to be any manual testing. I have done a quick look at the admin interface to ensure all looks in order.

Screen Shot 2023-08-03 at 10 00 49 am

Screen Shot 2023-08-03 at 9 58 15 am

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: app/webpacker/css/admin/variables.scss [hunk 1]
```diff
@@ -2,6 +2,6 @@
 $color-warning: $color-5;
 $color-tbl-border: $pale-blue;
 
-$color-modal-close-btn: #de6060;
-$color-modal-close-btn-hover: #bf4545;
+$color-modal-close-btn: $color-5;
+$color-modal-close-btn-hover: darken($color-5, 5%);
```
Comment (mkllnk) [inline app/webpacker/css/admin/variables.scss]: TIL about the darken function. :smile:

**Review by mkllnk — APPROVED** [review]
Great!

I reckon once the redesign is in production, we may want to have the front-end colours available for the editor styling instead of copying them. But it's a good move for now to keep it independent.

**Comment by RachL** [comment]
@mkllnk you've moved this one in test ready, but the label says no test?

**Comment by mkllnk** [comment]
Oops, I didn't look at the label. I'll merge it then. :smile:

**Comment by jibees** [comment]
wahoo, nice cleanup, thanks!

**Pull request merged**